types/react-big-calendar/react-big-calendar-tests.tsx

```diff
@@ -85,6 +85,26 @@ class CalendarResource {
     ReactDOM.render(<DnD localizer={localizer} />, document.body);
 }
 
+// overriding 'views' props
+{
+    const DaySFC: React.SFC = () => null;
+    class MonthView extends React.Component {
+        render() {
+            return null;
+        }
+    }
+    // supplying object to 'views' prop with only some of the supported views
```

Comment:
But then your defining a value for every view. It's not actually testing optionality of `views`

Reply:
@eps1lon - This case is testing that some `views` properties are optional, i.e. not all of them need to be supplied.  I've updated the comment to say so.

I also added a new test case demonstrating the optionality of `views`

Reply:
Sure but that wasn't the case at the time I made the comment. Now it's doing what the comment it says it does.